### DISCUSSION TITLE
How to set System Disk type and size with opentelekomcloud_compute_instance_v2

### DISCUSSION THREAD

**Discussion by agepi477** [opentelekomcloud/terraform-provider-opentelekomcloud#827] [Q&A] [answered]
Hi,

I'm new to terraform and OTC and I hope you guys can help me with this. I succesfully deployed an ECS VM with the newest version of your terraform provider and the opentelekomcloud_compute_instance_v2 resource and I wonder how to change the type and the size of the system volume.

Thanks in advance for your help.

**Comment by outcatcher** [comment, marked as answer]
Hi,

You can modify your disks configuration with `block_device` argument: [docs/resources/compute_instance_v2#boot-from-volume](https://registry.terraform.io/providers/opentelekomcloud/opentelekomcloud/latest/docs/resources/compute_instance_v2#boot-from-volume).

There are also several other disk setups described in this doc ― maybe you will find some of them useful.

I hope, I've answered your question :slightly_smiling_face:

**Reply by agepi477** [reply]
Thanks, I'll check that out. Thanks for your quick response and your help.
